forge-core/src/main/java/forge/deck/DeckFormat.java

```diff
@@ -60,43 +58,19 @@ public String getAttractionDeckConformanceProblem(Deck deck) {
         }
     },
     Commander      ( Range.is(99),                         Range.between(0, 10), 1, null,
-            card -> StaticData.instance().getCommanderPredicate().apply(card)
+            card -> StaticData.instance().getCommanderPredicate().apply(card), null
     ),
     Oathbreaker      ( Range.is(58),                         Range.between(0, 10), 1, null,
-            card -> StaticData.instance().getOathbreakerPredicate().apply(card)
+            card -> StaticData.instance().getOathbreakerPredicate().apply(card), null
     ),
     Pauper      ( Range.is(60),                         Range.between(0, 10), 1),
     Brawl      ( Range.is(59), Range.between(0, 15), 1, null,
-            card -> StaticData.instance().getBrawlPredicate().apply(card)
+            card -> StaticData.instance().getBrawlPredicate().apply(card), null
     ),
-    TinyLeaders    ( Range.is(49),                         Range.between(0, 10), 1, new Predicate<CardRules>() {
-        private final Set<String> bannedCards = ImmutableSet.of(
-                "Ancestral Recall", "Balance", "Black Lotus", "Black Vise", "Channel", "Chaos Orb", "Contract From Below", "Counterbalance", "Darkpact", "Demonic Attorney", "Demonic Tutor", "Earthcraft", "Edric, Spymaster of Trest", "Falling Star",
-                "Fastbond", "Flash", "Goblin Recruiter", "Grindstone", "Hermit Druid", "Imperial Seal", "Jeweled Bird", "Karakas", "Library of Alexandria", "Mana Crypt", "Mana Drain", "Mana Vault", "Metalworker", "Mind Twist", "Mishra's Workshop",
-                "Mox Emerald", "Mox Jet", "Mox Pearl", "Mox Ruby", "Mox Sapphire", "Najeela, the Blade Blossom", "Necropotence", "Shahrazad", "Skullclamp", "Sol Ring", "Strip Mine", "Survival of the Fittest", "Sword of Body and Mind", "Time Vault", "Time Walk", "Timetwister",
-                "Timmerian Fiends", "Tolarian Academy", "Umezawa's Jitte", "Vampiric Tutor", "Wheel of Fortune", "Yawgmoth's Will");
-
-        @Override
-        public boolean apply(CardRules rules) {
-            // Check for split cards explicitly, as using rules.getManaCost().getCMC()
-            // will return the sum of the costs, which is not what we want.
-            if (rules.getMainPart().getManaCost().getCMC() > 3) {
-                return false; //only cards with CMC less than 3 are allowed
-            }
-            ICardFace otherPart = rules.getOtherPart();
```

Comment:
whats happening to the other checks...?

Reply:
Whoops, that was sloppy of me... I was a bit hasty with trimming the old hardcoded banned cards and chopped that out by accident. Have added that check back into the predicate and tested that it is being used. Thank you for being vigilant in your code review!